skiko/buildSrc/src/main/kotlin/properties.kt

```diff
@@ -135,4 +135,8 @@ object SkikoArtifacts {
     val commonArtifactId = "skiko-jvm"
     fun runtimeArtifactIdFor(os: OS, arch: Arch) =
         "skiko-jvm-runtime-${targetId(os, arch)}"
+    fun nativeRuntimeArtifactIdFor(os: OS, arch: Arch) =
+        "skiko-native-runtime-${targetId(os, arch)}"
+    fun nativeSkiaInteropArtifactIdFor(os: OS, arch: Arch) =
```

Comment:
Why do we need second artifact? `skiko-native-runtime` shall be sufficient.

Reply:
Discussed offline. We have to have two artifacts for now.

Reply:
Ack